README.md

```diff
@@ -15,13 +15,15 @@ yarn add @vtex-components/accordion
 ```tsx
 import React, { Component } from 'react'
 
-import MyComponent from '@vtex-components/accordion'
-
-class Example extends Component {
-  render() {
-    return <MyComponent />
-  }
-}
+import Accordion from '@vtex-components/accordion'
+
+const Example = () => (
+  <Accordion>
+    <Accordion.Section header="Title">
+      This is the content
+    </Accordion.Section>
```

Comment:
can u add one more Section? This currently way, it looks like a Collapse and not an Accordion